sdk/identity/identity/test/manual-integration/AzureFunctions/INSTRUCTIONS.md

```diff
@@ -0,0 +1,116 @@
+# Testing Identity in Azure Functions
+
+## Configuration
+
+Before you begin, you must have the following:
+
+- An Azure account with an active subscription
+- The Azure Functions Core Tools version 3.x
+- Node.js, Active LTS version
+- One of the following tools for creating Azure resources:
+  - Azure CLI version 2.4 or later
+  - Azure PowerShell version 5.0 or later
+
+## Prerequisites
+
+Verify your prerequisites, which depend on whether you are using Azure CLI or Azure PowerShell for creating Azure resources
+
+For Azure CLI,
+
+1. In a terminal or command window, run `func --version` to check that the Azure Functions Core Tools are version 3.x.
+2. Run `az --version` to check that the Azure CLI version is 2.4 or later.
+3. Run `az login` to sign in to Azure and verify an active subscription.
+
+For Azure PowerShell,
+
+1. In a terminal or command window, run `func --version` to check that the Azure Functions Core Tools are version 3.x.
+2. Run `(Get-Module -ListAvailable Az).Version` and verify version 5.0 or later.
+3. Run `Connect-AzAccount` to sign in to Azure and verify an active subscription.
+
+### Run Locally
```

Comment:
To run locally you also need env vars - AZURE_CLIENT_SECRET, AZURE_CLIENT_ID, AZURE_TENANT_ID, and KEYVAULT URI - so we need to document how to set that up. 

Alternatively, there's no need to run this locally - only on azure functions - so we can omit this step entirely.